package.json

```
@@ -11,6 +13,35 @@
     "start": "node ./index.js",
     "test": "run-s lint jest"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud.git"
+  },
+  "keywords": [
+    "bitbucket",
+    "dependencies",
+    "dependency",
+    "docker",
+    "management",
+    "outdated",
+    "update"
```

Would these keywords make more sense? (same as Github repo)
```suggestion
    "bitbucket",
    "bitbucket-cloud",
    "bot",
    "renovate"
```
Maybe `automerge` or `autoapprove` or `approve` or `approver` (or combinations with hyphens) could be good

---

I will add `automerge` myself.